amgut/db/patches/0041.sql

```diff
@@ -236,17 +244,17 @@ CREATE TABLE pm.water_lot (
     CONSTRAINT uq_water_lot_name UNIQUE ( name )
  );
 
-CREATE TYPE target_region AS ENUM ('16S', '18S', 'ITS');
-CREATE TYPE target_subfragment AS ENUM ('V4');
+CREATE TYPE pm.target_region AS ENUM ('16S', '18S', 'ITS');
```

Comment:
Good question, I just fixed the pm part of this but I can add more ... @josenavas